bin/strata-cli/src/cmd/send.rs

```diff
@@ -35,25 +34,27 @@ pub struct SendArgs {
     /// address to send to
     #[argh(positional)]
     address: String,
+
+    /// override signet fee rate in sat/vbyte. must be >=1
+    #[argh(option)]
+    fee_rate: Option<u64>,
```

Comment:
You should add a check in https://github.com/alpenlabs/strata/blob/49b926e151bfe880a847058959d4708fc3f664cc/bin/strata-cli/src/signet.rs#L51-L59
to see if the `fee_rate` is `>=1`.

Or is this done internally in `FeeRate::from_sat_per_vb`?